packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/shared/use-create-new-entity-and-redirect.ts

```diff
@@ -158,15 +84,7 @@ export const useCreateNewEntityAndRedirect = () => {
         data: {
           entityTypeId: entityType.$id,
           ownedById,
-          /**
-           * @todo after implementing this ticket: https://app.asana.com/0/1203312852763953/1203433085114587/f (internal)
-           * we should just use `properties: {}` here, and delete `generateDefaultProperties` function,
-           * this is a temporary workaround for entity table to show the rows with empty values
-           */
-          properties: generateDefaultProperties(
-            entityType.properties,
-            subgraph,
-          ),
+          properties: {},
```

Comment:
Why are we creating an entity with empty properties? This is concerning to me. What if the entity type has required properties, will this code break the second we add validation?

Reply:
I think we can talk about "default properties" instead of "required properties" here. When we have "default values for properties", we can initialize the `properties` object using those. But I guess that's not priority right now. 

Entities can be an invalid state right? (e.g. missing required properties, invalid properties). Assuming we'll create the entity in draft state.

Reply:
Right, but default values wouldn't mean you have a valid entity. If I have a Person entity with a required "name", "age", and "location" field, and I have a default value for "location", I still wouldn't have a valid Person entity.

> Entities can be an invalid state right?

No. I'm sorry if we've been somewhat unclear on that, I know we said explicitly a while ago that we could do that to unblock development, because the backend does not have validation yet, but that's purely for short-term development speed. We should be building with the assumption that things actually are being validated.

Reply:
There is no concept of "draft" state